Resources/translations/fieldedit.en.xlf

```diff
@@ -857,6 +857,20 @@
         <note>key: value.should.be.valid.integer</note>
         <jms:reference-file>./Resources/public/js/views/fields/ez-integer-editview.js</jms:reference-file>
       </trans-unit>
+      <trans-unit id="d85e23987dc9ef6f162a6173ed81f647d9ff9fe1" resname="value.should.be.less.than.infinity">
+        <source>The value should be less than Infinity</source>
```

Comment:
As an editor I don't know what this error message is trying to tell me.

Unsure how to make it more clear while being specific, maybe simply "The value is to large, please use a valid number"


Same below